e2etest/bgptests/bgp.go

```diff
@@ -907,57 +936,87 @@ var _ = ginkgo.Describe("BGP", func() {
 							},
 						},
 					},
-				}}, ipfamily.IPv6))
+				}, {
+					Name:     "bgp-with-no-community",
+					Protocol: config.BGP,
+					Addresses: []string{
+						"fc00:f853:0ccd:e799::19-fc00:f853:0ccd:e799::26",
+					},
+				}},
+				"fc00:f853:ccd:e799::1",
+				"fc00:f853:ccd:e799::2",
+				"fc00:f853:ccd:e799::20",
+				ipfamily.IPv6))
 
 		table.DescribeTable("configure bgp local-preference and verify it gets propagated",
-			func(poolAddresses []string, ipFamily ipfamily.Family, localPref uint32) {
+			func(addressesWithPref, addressesNoPref, ipWithLocalPref, ipNoLocalPref string, family ipfamily.Family, localPref uint32) {
```

Comment:
Would it add value having 2 ips with LocalPref like you did with the communities?

Reply:
I just wanted to ensure the mechanism did not break with multiple rules, but this won't hurt and it won't change the tests running time.